specification/cost-management/resource-manager/readme.md

```diff
@@ -417,15 +417,33 @@ directive:
   - suppress: PathForPutOperation
     from: costallocation.json
     reason: 'Subscripiton and ResourceGroup scope is not supported in cost allocation by desgin'
+  - suppress: RequiredReadOnlySystemData
+    from: costallocation.json
+    reason: 'cost allocation does not return system data and will consider adding it in the future or upcoming api version'
+  - suppress: EnumInsteadOfBoolean
+    from: costallocation.json
+    reason: 'Keeping it as boolean property as per the design'
   - suppress: NoDuplicatePathsForScopeParameter
     from: settings.json
     reason: 'Settings does not use scope for List API'
   - suppress: GetCollectionOnlyHasValueAndNextLink
     from: settings.json
     reason: 'Settings List designed to deliver very limited records'
+  - suppress: PageableOperation
+    from: settings.json
+    reason: 'Settings List designed to deliver very limited records'
```

Comment:
In general, even if your RP doesn't return pages of records, ARM might want to convert your list response to a paged one for some reason (fanout routing, RBAC, etc), so you're recommended to add nextLink even when your RP doesn't return nextLinks itself.